uktrade/data-hub-frontend
# Add Lead adviser filter

## Description of change

As we can now allocate a Lead ITA to a company we need a filter that will filter companies by Lead ITA or Global Account manager.

## Test instructions

Go to the companies page, find the filter on the bottom left hand side of the screen and search for a Lead ITA/Global Account manager and you should see the results of all companies that adviser has been allocated too. 
 
## Screenshots
![Screenshot 2019-12-06 at 16 18 09](https://user-images.githubusercontent.com/10154302/70338022-07725e80-1844-11ea-9b56-6316f33cda79.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
